docs/resources/oauth/token.yml

```diff
@@ -0,0 +1,79 @@
+summary: |-
+  Request access token
+
+description: |-
+  Request an Access Token using either a client-side obtained OAuth 2.0
+  authorization code or a server-side JWT assertion.
+
+  An Access Token is a string that enables Box to verify that a
```

Comment:
`Box` what does it refer to? Is that from a generator 😅 ?

Reply:
🙈 , I'm refer the [Box-OpenAPI](https://github.com/box/box-openapi/blob/main/content/openapi.yml), so I copied that text from their docs 😭

Reply:
@olivierobert @hoangmirs I updated the docs in https://github.com/nimblehq/nimble-survey-web/pull/86/commits/41f9d6756709dadf7514804f5741ce1f2c34ca92